### PR TITLE
BaSP-TG-66: Shared DatePicker modal component

### PR DESCRIPTION
**Pull request by GonzaloAsencio**
Create a shared datepicker component .
Refactor in all the components where selects are being used with the new props structure.
Required props that we need to consider: disabled.

### CODE REVIEW

File: src/Components/Shared/Modal/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import styles from './modal.module.css';
+
+function Modal({
```
Comment (JuliMonte5) [inline src/Components/Shared/Modal/index.js:3]: Should be an arrow function

File: src/Components/Shared/Modal/modal.module.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    top: 0;
+    right: 0;
+    bottom: 0;
+    position: fixed;
+    justify-content: center;
+    align-items: center;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
```
Comment (JuliMonte5) [inline src/Components/Shared/Modal/modal.module.css:12]: Duplicated properties

**Review by JuliMonte5 — CHANGES_REQUESTED** [review]

File: src/Components/Employees/Employee.js [hunk 3]
```diff
@@ -27,12 +35,11 @@ const Employee = ({ employee, onDeleteItem }) => {
...
         <td>
           <Modal
-            openModal={showModal}
+            showModal={showModal}
             closeModal={closeModal}
-            deleteAction={deleteEmployee}
-            id={employee._id}
-            warningText={`Are you sure you want to remove Employee: ${employee.name}`}
+            confirmAction={deleteEmployee}
             title={'DELETE EMPLOYEE'}
```
Comment (ignacioaristo) [inline src/Components/Employees/Employee.js:41]: move those message in a variable outside

File: src/Components/Projects/Form/index.js [hunk 4]
```diff
@@ -95,13 +96,11 @@ const index = (props) => {
         body: JSON.stringify(projectBody)
       });
       const data = await response.json();
-
-      if (response.status === 200) {
-        alert(data.msg);
+      //CAMBIAR ESTADO  200 por 201
```
Comment (ignacioaristo) [inline src/Components/Projects/Form/index.js:99]: remove this comment

File: src/Components/Projects/List/index.js [hunk 5]
```diff
@@ -55,12 +55,11 @@ const ProjectList = ({ projectItem, onDeleteItem }) => {
...
       <Modal
-        show={showModal}
+        showModal={showModal}
         closeModal={closeModal}
-        onDelete={deleteItem}
-        id={projectItem._id}
-        title={'Delete Project?'}
-        text={`Are you sure you want to delete project "${projectItem.name}"?`}
+        confirmAction={deleteItem}
+        title={'DELETE PROJECT'}
```
Comment (ignacioaristo) [inline src/Components/Projects/List/index.js:61]: move those message in a variable outside

**Review by ignacioaristo — CHANGES_REQUESTED** [review]

File: src/Components/Shared/Modal/index.js (hunk 1 above)
Comment (GonzaloAsencio) [reply src/Components/Shared/Modal/index.js:3]: Done

File: src/Components/Shared/Modal/modal.module.css (hunk 2 above)
Comment (GonzaloAsencio) [reply src/Components/Shared/Modal/modal.module.css:12]: Done

File: src/Components/Projects/Form/index.js (hunk 4 above)
Comment (GonzaloAsencio) [reply src/Components/Projects/Form/index.js:99]: Done

**Review by fede230298 — APPROVED** [review]

File: src/Components/Shared/Datepicker/index.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+const DatePicker = ({ changeValue, inputValue, label }) => {
+  const HandleChange = (e) => {
+    changeValue(e.target.value);
+  };
+
+  return (
+    <>
+      <label>{label}</label>
+      <input type="date" value={inputValue} onChange={(e) => HandleChange(e)}></input>
```
Comment (Ignacio-Sormani) [inline src/Components/Shared/Datepicker/index.js:9]: inputs need a name, that is sent through props, so it is easier to use it as a key when making a setState

**Review by Ignacio-Sormani — CHANGES_REQUESTED** [review]

**Review by JuliMonte5 — APPROVED** [review]

**Review by fede230298 — APPROVED** [review]

**Review by Ignacio-Sormani — APPROVED** [review]

**Pull request merged**